.gitignore

```diff
@@ -43,6 +43,5 @@ html_docs
 
 # random old stuff that we should look at the necessity of...
 /tmp/
-backwards/
```

Comment:
@s1monw I pushed another commit updating gitignore to be more explicit about the backwords folder (rather than using a global glob pattern). I also updated the comments around it.